server/planning/events/events.py

```diff
@@ -189,41 +189,46 @@ def on_create(self, docs):
         if generated_events:
             docs.extend(generated_events)
 
-    def validate_event(self, event):
+    def validate_event(self, updates, original=None):
         """Validate the event
 
         @:param dict event: event created or updated
         """
-        self._validate_multiday_event_duration(event)
-        self._validate_dates(event)
+        self._validate_multiday_event_duration(updates)
+        self._validate_dates(updates, original)
 
-        if len(event.get('calendars', [])) > 0:
+        if len(updates.get('calendars', [])) > 0:
             existing_calendars = get_resource_service('vocabularies').find_one(req=None, _id='event_calendars')
-            for calendar in event['calendars']:
+            for calendar in updates['calendars']:
```

Comment:
Comment this out, as the client interface removes disabled Calendars from the selection (and the qcodes must exist in that selection as they came from the CV in the first place).